src/testmod/java/dev/emi/trinkets/TestTrinket.java

```diff
@@ -1,10 +1,27 @@
 package dev.emi.trinkets;
 
+import com.google.common.collect.Multimap;
 import dev.emi.trinkets.api.TrinketItem;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.attribute.EntityAttribute;
+import net.minecraft.entity.attribute.EntityAttributeModifier;
+import net.minecraft.entity.attribute.EntityAttributes;
+import net.minecraft.item.ItemStack;
+
+import java.util.UUID;
 
 public class TestTrinket extends TrinketItem {
 
+	private static final EntityAttributeModifier SPEED_BOOST_MODIFIER = new EntityAttributeModifier(UUID.fromString("ac7ab816-2b08-46b6-879d-e5dea34ff305"), "trinkets-testmod:movement_speed", 0.4, EntityAttributeModifier.Operation.MULTIPLY_TOTAL);
+
 	public TestTrinket(Settings settings) {
 		super(settings);
 	}
+
+	@Override
+	public Multimap<EntityAttribute, EntityAttributeModifier> getModifiers(ItemStack stack, SlotReference slot, LivingEntity entity, UUID uuid) {
```

Comment:
This UUID behavior may be a holdover from experience you may have in curios, but that's actually a behavior trinkets corrects, you should be constructing an `EntityAttributeModifier` with the passed UUID.

Explanation if you want to know:
Using a constant UUID means that your effect won't stack, and only apply once, worse, if you wear 2 of the same trinket and then remove one, you will not get the effect at all (this is because unequiping removes the EAM and minecraft tracks EAMs using UUIDs). There are also some other edge cases that will result in unintended behavior. This is the sort of thing you don't have to worry about with armor slots or held items because only a single instance can be applied at once.

Reply:
Oh, that makes a lot of sense, thanks for the explanation.

Reply:
> This UUID behavior may be a holdover from experience you may have in curios

Also correct on this btw 😉

Reply:
I think this is less a holdover from any Curios experience (although it certainly doesn't help) and more to do with how modders are accustomed to setting up their EAMs in static initializers since vanilla doesn't need this sort of consideration. While I worry this will happen fairly often, I'm not sure how to make this more clear to modders since the javadocs are quite clear on the usage of the UUID parameter.